src/cets.erl

```diff
@@ -24,6 +24,8 @@
     insert_many/2,
     delete/2,
     delete_many/2,
+    delete_object/2,
+    delete_object_many/2,
```

Comment:
Please, please name it just `delete_objects`

Reply:
done